src/Commands/Site/CreateCommand.php

```diff
@@ -72,7 +72,16 @@ public function create($site_name, $label, $upstream_id, $options = ['org' => nu
         if ($site = $this->getSiteById($workflow->get('waiting_for_task')->site_id)) {
             $this->log()->notice('Deploying CMS...');
             $this->processWorkflow($site->deployProduct($upstream->id));
-            $this->log()->notice('Deployed CMS');
+            $this->log()->notice('Waiting for site availability...');
+            $waits = 0;
+            do {
+                sleep(5);
+                $woke = $site->getEnvironments()->get('dev')->wake();
+                // if success is empty, then the site is still waking up.
+                $waits++;
```

Comment:
Shouldn't log that everything is okay if `wake` doesn't get a valid result. Should exit early via a thrown exception if timeout is exceeded.

Reply:
Fixed.